Makefile

```diff
@@ -138,13 +138,17 @@ doc-setup: client
 	rm -Rf doc/.sphinx/.doctrees
 
 .PHONY: doc
-doc: doc-setup doc-incremental
+doc: doc-setup doc-incremental doc-objects
 
 .PHONY: doc-incremental
 doc-incremental:
 	@echo "Build the documentation"
 	. $(SPHINXENV) ; LOCAL_SPHINX_BUILD=True sphinx-build -c doc/ -b dirhtml doc/ doc/html/ -d doc/.sphinx/.doctrees -w doc/.sphinx/warnings.txt
 
+.PHONY: doc-objects
+doc-objects:
+	. $(SPHINXENV); cd doc/html; python3 -m sphinx.ext.intersphinx 'objects.inv' > objects.inv.txt
```

Comment:
I assume the resulting objects.inv.txt will be shipped along the docs -- just like the objects.inv file. Is that correct?

Reply:
That's the plan - but I just noticed that this will only work for the local docs, not the online version (because RTD doesn't use the Makefile). Let me add it for RTD as well ...

Reply:
Working now: https://canonical-ubuntu-documentation-library--12701.com.readthedocs.build/lxd/en/12701/objects.inv.txt